README.md

```diff
@@ -96,7 +96,7 @@ Translations of curriculum codes (sv: ämneskoder på schemat) to clear text des
 
 #### hooks
 
-To make it easier to use the the api in the app, we also created a set of React hooks.
+To make it easier to use the api in the app, we also created a set of React hooks.
```

Comment:
This is a new thing that I'm experimenting with, it seems to have done the right thing here.

Reply:
It sure does!